.github/workflows/release.yaml

```diff
@@ -0,0 +1,84 @@
+name: Release
+
+on:
+    workflow_dispatch:
+        inputs:
+            release-version:
+              type: string
+              required: true
+              description: 'Version number'
+              default: '0.1.0'
```

Comment:
Should we have a default version at all? It seems like it should never be used (we never want to release `0.1.0` again), so if it is ever used something has gone wrong